### PR TITLE
restructured

### PR DESCRIPTION
**Pull request by manasavesala**
please review @ajaykalangi

### CODE REVIEW

File: LinkedLists/AddTwoNumbers.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-class MainClass {
+class AddTwoNumbers {
```
Comment (ajaykalangi) [inline LinkedLists/AddTwoNumbers.cs:4]: Write a couple of lines that describe what this class is doing. preferably by showing an example

File: LinkedLists/AddTwoNumbers.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   public static void Main (string[] args) {
     Console.WriteLine ("Hello World");
```
Comment (ajaykalangi) [inline LinkedLists/AddTwoNumbers.cs:6]: Do you need this?

**Review by ajaykalangi — COMMENTED** [review]
Needs work

**Comment by manasavesala** [comment]
new code

File: Stacks/ValidateParaentheses.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+
+ class ValidatingParaentheses{
```
Comment (ajaykalangi) [inline Stacks/ValidateParaentheses.cs:4]: Add a comment describing what this does. Preferably with examples

File: Stacks/ValidateParaentheses.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  public static void Main(string[] args){
+    Console.WriteLine(validate(""));
+  }
+
+  private static bool validate(string str){
+    Stack s = new Stack();
```
Comment (ajaykalangi) [inline Stacks/ValidateParaentheses.cs:18]: It is a good idea to include what type of Stack it is, is this a stack of integers or characters or something else?

File: Stacks/ValidateParaentheses.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+
+ class ValidatingParaentheses{
+  public static void Main(string[] args){
+    Console.WriteLine(validate(""));
+  }
+
+  private static bool validate(string str){
```
Comment (ajaykalangi) [inline Stacks/ValidateParaentheses.cs:16]: What happens when input is null?

File: Stacks/ValidateParaentheses.cs [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    Stack s = new Stack();
+      for(int i=0; i<str.Length; i++){
+        if(str[i] == '('){
+          s.Push(str[i]);
+        }
+        if(str[i] == ')'){
+          try{
+            s.Pop();
+          }catch(Exception e){
```
Comment (ajaykalangi) [inline Stacks/ValidateParaentheses.cs]: Add a comment to signify what this Exception means, change this to specific exception that we would want to catch

File: Stacks/ValidateParaentheses.cs [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+            return false;
+          }
+        }
+      }
+
+      return s.Count == 0;
```
Comment (ajaykalangi) [inline Stacks/ValidateParaentheses.cs:35]: Add a comment on why this is what we expect to return

**Review by ajaykalangi — COMMENTED** [review]
Mostly comments on code

**Review by ajaykalangi — APPROVED** [review]

**Comment by ajaykalangi** [comment]
Great job!

**Pull request merged**